docs/notebooks/markov-models.ipynb

```diff
@@ -137,7 +137,7 @@
     "The symbol etymology, which usually gets swept under the rug in mathematically-oriented papers, are as follows: \n",
```

Comment:
<ul><li>"...which is exactly <s>what</s> WHERE the mean of the Gaussian..."</li><li>Here, the explanation about the state information given by the mean and variance of obs is clearer. But when <code>k != 0</code>  I don't see why the location doesn't also give you information about the state.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ericmjl/bayesian-analysis-recipes/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Good catch on the grammar. </p><p> </p><p>As for your question, I think the explanation is as follows: Regardless of whether k = 0 or not, by how we've structured the autoregressive dependency, the location doesn't depend on the state, so by definition, the location cannot give information about the state. Looking at it from the other perspective, if we're in a state that gives us high variance, then the previous emission can influence where we jump off from, but the collective average of locations over a given time don't tell us anything about the state.</p><p> </p><p>Overall, I think it takes playing with the model to see the point. Maybe I should phrase it as "collective average of locations over time".</p>

Reply:
I get it now, thanks! The mathematical explanation is actually very clear to me 😉